web-components/icon/library.default.ts

```diff
@@ -0,0 +1,11 @@
+import { IconLibrary, registerLibrary } from "./iconLibrary";
+
+let defaultIconsPathBase = './icons';
+export const setDefaultIconsPath = (iconsPath: string) => defaultIconsPathBase = iconsPath;
```

Comment:
How is this meant to be used?

Reply:
Basically, you can use it to set where the icons are served from. I.e. if we wanted to serve them from `chrome://resources/leo-icons` we could call `setDefaultIconsPath('chrome://resources/leo-icons')` and it would fetch the `atm` icon from `chrome://resources/leo-icons/atm.svg`